templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,12 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 59**
+
+[`PATCH /settings/display`](/api/update-display-settings): Added
+  new `timezone_auto_update` field to check if the user wants to
+  update his timezone automatically with the browser's timezone.
+
```

Comment:
This should be squashed into the commit adding the setting.

Reply:
Done